### ISSUE TITLE
It seems to have no effect on this work.

### ISSUE THREAD

**Issue by MoonWolf9067** [joe-siyuan-qiao/WeightStandardization#11]
@joe-siyuan-qiao Hi,thanks for your work! When I use the following code in my work, it seems to have no effect . Before the convolution, I made changes to the convolution kernel according to the following code.Then the convolution result is sent to the group normalization. So, can you give me some ideas? Thanks
![tg](https://user-images.githubusercontent.com/22995934/63774715-6acd2600-c910-11e9-9607-e640e0b09493.png)

**Comment by csrhddlam** [comment]
Hi, thanks for your interest! It is a little tricky to implement WS in Tensorflow. You need to make sure that WS is executed correctly in both the forward and the backward pass. If you're using the latest version, we recommend adding those lines before [this line](https://github.com/tensorflow/tensorflow/blob/r1.14/tensorflow/python/keras/layers/convolutional.py#L196). We also have an example of using WS in [dgcnn](https://github.com/csrhddlam/dgcnn/blob/master/utils/tf_util.py#L131). Moreover, you could use [tf.custom_gradient](https://www.tensorflow.org/api_docs/python/tf/custom_gradient) and/or [tf.print](https://www.tensorflow.org/api_docs/python/tf/print) to further verify your implementation. Thanks.

Huiyu
